internal/k8s/client.go

```diff
@@ -41,6 +42,7 @@ import (
 )
 
 type Namespace string
+type NamespaceFlag string
```

Comment:
Minor, but I think for clarity it might make sense to call this `NamespaceOverride` (the fact that it comes from a flag when run from CLI is sorta incidental)